sw/device/lib/testing/test_framework/ottf_main.h

```diff
@@ -45,4 +52,58 @@ extern bool manufacturer_pre_test_hook(void);
  */
 extern bool manufacturer_post_test_hook(void);
 
+/**
+ * Create a FreeRTOS task.
+ *
+ * Tasks should be implemented as functions that never return. However, they may
+ * delete themselves using the `ottf_task_delete_self()` function defined below.
+ *
+ * Additionally, tasks are always run at a priority level higher than that of
+ * the FreeRTOS idle task's (which is a priority of 0).
+ *
+ * See the FreeRTOS `xTaskCreate` documentation for more details:
+ * https://www.freertos.org/a00125.html.
+ *
+ * @param task_function The name of the function that implements the task.
+ * @param task_name A task identification string used to help debugging.
+ * @param task_stack_depth The amount of memory to reserve for the task's stack.
+ * @param task_priority The numerical priority of the task.
+ * @return A boolean encoding the success of the operation.
+ */
+inline bool ottf_task_create(TaskFunction_t task_function,
+                             const char *task_name,
+                             configSTACK_DEPTH_TYPE task_stack_depth,
+                             uint32_t task_priority) {
+  return xTaskCreate(task_function, task_name, task_stack_depth, NULL,
```

Comment:
nit: from style guide, add `/*param_name=*/NULL` to all value parameters.

Reply:
fixed.